Fail to compile on mingw32 gcc 7.2

Master branch doesn't compile on msys2/mingw32, with gcc 7.2:

[ 12%] Building CXX object src/osvr/Common/CMakeFiles/osvrCommon.dir/AliasProcessor.cpp.obj
In file included from C:/build/OSVR-Core/build/src/osvr/Util/Verbosity.h:43:0,
                 from C:\build\OSVR-Core\src\osvr\Common\AliasProcessor.cpp:32:
C:/build/OSVR-Core/inc/osvr/Util/Logger.h:206:49: error: function 'const string& osvr::util::log::Logger::getName() const' definition is marked dllimport
             OSVR_UTIL_EXPORT std::string const &getName() const {
                                                 ^~~~~~~
mingw32-make[2]: *** [src\osvr\Common\CMakeFiles\osvrCommon.dir\build.make:88: src/osvr/Common/CMakeFiles/osvrCommon.dir/AliasProcessor.cpp.obj] Error 1
mingw32-make[1]: *** [CMakeFiles\Makefile2:1729: src/osvr/Common/CMakeFiles/osvrCommon.dir/all] Error 2
mingw32-make: *** [Makefile:162: all] Error 2

--- a/inc/osvr/Util/LogRegistry.h
+++ b/inc/osvr/Util/LogRegistry.h
@@ -58,10 +58,10 @@ namespace util {
 
         class LogRegistry {
           public:
-            OSVR_UTIL_EXPORT LogRegistry(LogRegistry const &) = delete; // copy construct
-            OSVR_UTIL_EXPORT LogRegistry(LogRegistry &&) = delete;      // move construct
-            OSVR_UTIL_EXPORT LogRegistry &operator=(LogRegistry const &) = delete; // copy assign
-            OSVR_UTIL_EXPORT LogRegistry &operator=(LogRegistry &&) = delete;      // move assign
+            LogRegistry(LogRegistry const &) = delete; // copy construct
+            LogRegistry(LogRegistry &&) = delete;      // move construct
+            LogRegistry &operator=(LogRegistry const &) = delete; // copy assign
+            LogRegistry &operator=(LogRegistry &&) = delete;      // move assign
 
             OSVR_UTIL_EXPORT static LogRegistry &instance(std::string const * = nullptr);
 
@@ -115,11 +115,11 @@ namespace util {
              */
             OSVR_UTIL_EXPORT void setConsoleLevel(LogLevel severity);
 
-            OSVR_UTIL_EXPORT std::string const &getLogFileBaseName() const {
+            std::string const &getLogFileBaseName() const {
                 return logFileBaseName_;
             }
 
-            OSVR_UTIL_EXPORT bool couldOpenLogFile() const { return sinks_.size() > 1; }
+            bool couldOpenLogFile() const { return sinks_.size() > 1; }
 
           protected:
             OSVR_UTIL_EXPORT LogRegistry(std::string const &logFileBaseName);

--- a/inc/osvr/Util/Logger.h
+++ b/inc/osvr/Util/Logger.h
@@ -101,10 +101,10 @@ namespace util {
                           spdlog::sinks_init_list sinks);
 
             /// Non-copyable
-            OSVR_UTIL_EXPORT Logger(const Logger &) = delete;
+            Logger(const Logger &) = delete;
 
             /// Non-copy-assignable
-            OSVR_UTIL_EXPORT Logger &operator=(const Logger &) = delete;
+            Logger &operator=(const Logger &) = delete;
 
             /// Destructor
             OSVR_UTIL_EXPORT ~Logger();
@@ -203,7 +203,7 @@ namespace util {
             OSVR_UTIL_EXPORT void flush();
 
             /// Get the logger name
-            OSVR_UTIL_EXPORT std::string const &getName() const {
+            std::string const &getName() const {
                 return name_;
             }
 

